content/en/examples/admin/resource/memory-available-cgroupv2.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
This second line doesn't make sense

Reply:
Thanks, removed it. I based it on https://kubernetes.io/examples/admin/resource/memory-available.sh